projekte/2016-08-15-karlsruhe-drl-schauhin.html

```diff
@@ -1,14 +1,23 @@
 ---
 layout: project #do not change
 lab: OK Lab Karlsruhe #change into your city/lab
-imgname: #add file to assets/projects/your city
+imgname: karlsruhe/schau-hin-website.png
```

Comment:
Image is missing

Reply:
Uploaded the picture now, thanks for pointing it out